apps/experiments/views/experiment.py

```diff
@@ -231,24 +247,59 @@ class CreateExperiment(BaseExperimentView, CreateView):
     button_title = "Create"
     permission_required = "experiments.add_experiment"
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+        if "file_formset" not in context:
```

Comment:
when would "file_formset" already be in the context?

Reply:
Oh wait I think it would be when there's some error and we return the invalid form. Nevermind, I should have read further